spec/support/api/claims/advocate_hardship_claim_test.rb

```diff
@@ -0,0 +1,89 @@
+require_relative 'base_claim_test'
```

Comment:
This whole file seems to have leaked over from the AGFS branch, not a problem unless it changes before the merge

Reply:
👍 have removed redundant file (it is in agfs branch).